src/parsers/pm_io/common.jl

```diff
@@ -34,7 +34,7 @@ end
 ""
 function parse_json(io::IO; validate=true)
     data_string = read(io, String)
-    pm_data = JSON.parse(data_string)
+    pm_data = JSON2.parse(data_string)
```

Comment:
and this

Reply:
I just ran the tests and everything ran. These lines probably never get called.

Reply:
They might be correct. I think you should confirm that it generates the same result before pushing.

Reply:
You are correct, this doesn't return the same thing.